README.md

```diff
@@ -0,0 +1,262 @@
+[basics]: https://quantumai.google/cirq/start/basics
+[Build]: https://quantumai.google/cirq/build
+[Simulate]: https://quantumai.google/cirq/simula
+
+### Reference Documentation
+
+*   Docs for the [current stable release] correspond to what you get with
+    `pip install cirq`.
+*   Docs for the [pre-release] correspond to what you get with
+    `pip install cirq~=1.0.dev`.
+
+[current stable release]: https://quantumai.google/reference/python/cirq/all_symbols
+[pre-release]: https://quantumai.google/reference/python/cirq/all_symbols?version=nightly
+
+### Examples
+
+*   The [examples subdirectory](./examples/) of the Cirq GitHub repo has many
+    programs illustrating the application of Cirq to everything from common
+    textbook algorithms to more advanced methods.
+*   The [Experiments page](https://quantumai.google/cirq/experiments/) on the
+    Cirq documentation site has yet more examples, from simple to advanced.
+
+### Change log
+
+*   The [Cirq releases](https://github.com/quantumlib/cirq/releases) page on
+    GitHub lists the changes in each release.
+
+## Integrations
+
+Google Quantum AI has a suite of open-source software that lets you do more
+with Cirq. From high-performance simulators, to novel tools for expressing and
+analyzing fault-tolerant quantum algorithms, our software stack lets you
+develop quantum programs for a variety of applications.
+
+<div align="center">
+
+| Your interests                                  | Software to explore  |
+|-------------------------------------------------|----------------------|
+| Quantum algorithms?<br>Fault-tolerant quantum computing (FTQC)? | [Qualtran] |
+| Large circuits and/or a lot of simulations?     | [qsim] |
+| Circuits with thousands of qubits and millions of Clifford operations? | [Stim] |
+| Quantum error correction (QEC)?                 | [Stim] |
+| Chemistry and/or material science?              | [OpenFermion]<br>[OpenFermion-FQE]<br>[OpenFermion-PySCF]<br>[OpenFermion-Psi4] |
+| Quantum machine learning (QML)?                 | [TensorFlow Quantum] |
+| Real experiments using Cirq?                    | [ReCirq] |
+
+</div>
+
+[Qualtran]: https://github.com/quantumlib/qualtran
+[qsim]: https://github.com/quantumlib/qsim
+[Stim]: https://github.com/quantumlib/ssim
+[OpenFermion]: https://github.com/quantumlib/openfermion
+[OpenFermion-FQE]: https://github.com/quantumlib/OpenFermion-FQE
+[OpenFermion-PySCF]: https://github.com/quantumlib/OpenFermion-PySCF
+[OpenFermion-Psi4]: https://github.com/quantumlib/OpenFermion-Psi4
+[TensorFlow Quantum]: https://github.com/tensorflow/quantum
+[ReCirq]: https://github.com/quantumlib/ReCirq
+
+## Community
+
+<a href="https://github.com/quantumlib/Cirq/graphs/contributors"><img
+align="right" width="160em" alt="Total number of contributors to Cirq"
+src="https://img.shields.io/github/contributors/quantumlib/cirq?label=Contributors&logoColor=white&logo=github&color=cccccc&style=flat-square"/></a>
```

Comment:
Personally I did not notice the Contributors badge on the RHS until reading this source.
Please consider keeping it simpler here and just adding the badge just above the "Cirq has benefited..." text.

Reply:
Ah, okay, so not a good placement. 

The problem I was trying to solve is that the link to the contributor graph only shows the first 100 contributors, which, for casual readers, might make our "over 200 contributors" statement seem suspect. It seems that the only way to get the actual number is via API, and the only way I could quickly find to get the value into the page is to use a badge. This is why the contributors badge is showing up again, after our previous discussion in the earlier PR about removing this badge and the stars badge. I was trying to make it unobtrusive.

Anyway, I'll reposition it.